Sources/ParseSwift/Authentication/3rd Party/ParseFacebook.swift

```diff
@@ -44,19 +44,19 @@ public struct ParseFacebook<AuthenticatedUser: ParseUser>: ParseAuthentication {
                             authenticationToken: String?,
                             expiresIn: Int? = nil) -> [String: String] {
 
-            var returnDictionary = [AuthenticationKeys.id.rawValue: userId]
+            var returnDictionary = [CodingKeys.id.rawValue: userId]
```

Comment:
Please revert all of these back to AuthenticationKeys

Reply:
The actual fix should be applied to lines 25-27:

```swift
enum AuthenticationKeys: String, Codable {
        case id // swiftlint:disable:this identifier_name
        case authenticationToken = "token"
        case accessToken = "access_token"
        case expirationDate = "expiration_date"
```

You can deleted the CodingKeys, 29-34

Reply:
Can you also make the similar change to Twitter: https://github.com/parse-community/Parse-Swift/blob/252df53eee6e8f95178eb3f58ce7124ab5673b3a/Sources/ParseSwift/Authentication/3rd%20Party/ParseTwitter.swift#L27-L39